packages/contracts/contracts/L1/interfaces/IAssetManager.sol

```diff
@@ -507,9 +507,9 @@ interface IAssetManager {
      * @notice Undelegate the KRO of given shares for the given validator.
```

Comment:
Please modify here also!

Reply:
Sorry about that. I guess I just wanted to merge quickly...